boilerplate/app/models/Episode.ts

```diff
@@ -9,22 +18,31 @@ export const EpisodeModel = types
   .props({
     guid: types.identifier,
     title: "",
-    pubDate: "",
+    pubDate: "", // Ex: 2022-08-12 21:05:36
     link: "",
     author: "",
     thumbnail: "",
     description: "",
     content: "",
-    enclosure: types.frozen(),
+    enclosure: types.frozen<Enclosure>(),
     categories: types.array(types.string),
-
-    // additional properties
-    favorite: false,
   })
   .actions(withSetPropAction)
   .views((episode) => ({
     get datePublished() {
-      return new Date(episode.pubDate)
+      const date = new Date(episode.pubDate)
+      return format(date, "MMM dd, yyyy")
```

Comment:
I don't know if we want to add `date-fns` just for this. Do you think it is widely used enough to validate including it in the base boilerplate?

Reply:
I've used it on nearly ever client project

Reply:
I'm open to alternatives though

Reply:
Moment is also a huge import. `date-fns` is good or `luxon` is another alternative (I think by an ex-moment team member that realized the pitfalls of it). 

Not sure if @mazenchami or @leonskim want to weigh in from an International support perspective as well.

Reply:
Let's include date-fns ... thanks @robinheinze !

Reply:
great question @frankcalise. I know they both have internationalization, but not sure if Arabic or Korean are part of it already. Would be worth testing that part when setting this up

Reply:
documentation for `date-fns/i18n`  [here](https://date-fns.org/v2.29.2/docs/I18n)